src/client/datascience/jupyter/kernels/cellExecution.ts

```diff
@@ -252,72 +229,16 @@ export class CellExecution {
      */
     private dispose() {
         traceCellMessage(this.cell, 'Execution disposed');
-        this.activeExecution = undefined;
         this.disposables.forEach((d) => d.dispose());
         const deferred = CellExecution.cellsCompletedForTesting.get(this.cell);
         if (deferred) {
             deferred.resolve();
         }
     }
 
-    private async interruptExecution(
```

Comment:
Moved to KenrelExecution, else we ended up with multiple interrupts, one for each cell.